resources/views/OrderForm/OrderForm.tsx

```diff
@@ -0,0 +1,107 @@
+
+import {Col, Row, Typography, Button, InputNumber, Divider} from 'antd'
+import React, {useState, useEffect, ReactNodeArray}  from 'react'
+import {StoreType, DispatchArgumentType} from '@types'
+import {pathToRegexp} from 'path-to-regexp'
+import {connect} from 'react-redux'
+import {OrderItemStateType} from 'models/order'
+import {getInventoryDetail} from 'services/inventory'
+
+type OrderFormPropType = {
+    dispatch: (arg: DispatchArgumentType) => void,
+    OrderItem: OrderItemStateType,
+  }
+
+const OrderForm = ({
+    dispatch,
+    OrderItem
+  }: OrderFormPropType) => {
+    const {Text} = Typography
+    const {id, data} = OrderItem
+    const [quantity, setQuantity] = useState<number>(1)
+    const [total, setTotal] = useState<number>(0)
+
+    const fetchOrderDetail = () => {
+      getInventoryDetail(id).then((response: any) => {
+          if (response.data.success) {
+              dispatch({type: 'SET_ORDER_DETAIL_DATA', payload: response.data.data})
+          }
+      })
+    }
+
+    useEffect(() => {
+      const regexp = pathToRegexp('/OrderForm/(\\d+)')
+      const IdString = regexp.exec(window.location.pathname)
+      const pathId = IdString && parseInt(IdString[1])
+      dispatch({type: 'SET_ITEM_DETAIL_ID', payload: pathId})
+
+      if (id !== 0 ) {
+        fetchOrderDetail()
+    }
+    }, [id])
+    const ignoredKeys = ['id', 'created_at' , 'updated_at', 'cost', 'stock']
```

Comment:
i think we can turn this few lines of code into a reusable function for data display, then add jest unit test to it